gems/pending/spec/postgres_ha_admin/database_yml_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'postgres_ha_admin/database_yml'
+
+describe PostgresHaAdmin::DatabaseYml do
+  let(:yml_utils) { described_class.new(@yml_file.path, 'test') }
+
+  before do
+    @yml_file = Tempfile.new('database.yml')
+    yml_data = YAML.load('---
+      base:
+        username: user
+        wait_timeout: 5
+        port:
+      test:
+        pool: 3
+        database: vmdb_test'
+                        )
+    File.write(@yml_file.path, yml_data.to_yaml)
+  end
+
+  after do
+    @yml_file.close(true)
+  end
+
+  describe "#pg_params_from_database_yml" do
+    it "returns pg connection parameters based on 'database.yml'" do
+      params = yml_utils.pg_params_from_database_yml
+      expect(params).to include(:dbname => 'vmdb_test', :user => 'user')
```

Comment:
make sense, instead of two expectation (include and not_to include) use just one with exact match